### ISSUE TITLE
[Elixir] Implement new Concept Exercise: `file`

### ISSUE THREAD

**Issue by angelikatyborska** [exercism/v3#2802]
## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student how to read from files and how to write to files. It is also required as a prerequisite for the `rpn-calcualtor-output` concept exercise.

## Learning objectives

- Know about the `File` module
- Know about the easy-to-use `File.read` and `File.write` functions
- Know about the more-detailed `File.open`, `File.close` and how to use the `IO` module together with those
- Know about some more common modes when opening a file, e.g. `:read`, `:write`, `:append`
- Know that opening a file starts a new process (https://hexdocs.pm/elixir/File.html#module-processes-and-raw-files)

## Out of scope

?

## Concepts

- `file`

## Prerequisites

- `io`
- `processes`
- `pids`
- something else, depending on the story

## Story

Look through existing stories (`/reference/stories/`) to find inspiration or an existing story that can work for this exercise, or come up with a totally original idea.

## Implementing

We can use actual files in this exercise. Remember to keep all paths relative to the directory with the exercise.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.
